content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -498,6 +593,14 @@ The Account Resource supports proprietary systems for:
 
 * Account.identifier.system is [Account Number](#account-number) for financial accounts.
 
+### Medications
+
+<h4>MedicationRequest</h4>
+
+The MedicationRequest Resource supports proprietary systems for:
+
+* MedicationRequest.category is [Account Number](#account-number) for financial accounts.
```

Comment:
Need to update the description

Reply:
https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce?file-filters%5B%5D=.md#diff-6fdcb00b5b1b6edd51b687794bea6b95R602